bbb-learning-dashboard/src/components/QuizzesChart.jsx

```
@@ -0,0 +1,149 @@
+          <Tooltip
+            cursor={{ strokeDasharray: '3 3' }}
+            content={(tooltipProps) => {
+              const { active, payload } = tooltipProps;
+              const isVisible = active && payload?.length;
+              return (
+                <Paper style={{ visibility: isVisible ? 'visible' : 'hidden' }} className="p-2">
+                  {isVisible && (
+                    <>
+                      <p className="font-bold">
+                        {[payload[0].payload.names.map((name) => (
+                          <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)